src/libPMacc/CMakeLists.txt

```diff
@@ -56,5 +56,5 @@ target_link_libraries(check ${LIBS})
 enable_testing()
 add_test(PMacc_test_build "${CMAKE_COMMAND}" --build ${CMAKE_BINARY_DIR} --target check)
 # --alow-run-as-root because some ci run test as root
-add_test(PMacc_test_run  mpiexec --allow-run-as-root -n 2 check )
+add_test(PMacc_test_run  mpiexec -n 1 ./check --log_level=test_suite)
```

Comment:
Cmake had some problems to call mpiexec with these arguments on the hypnos cluster. I will remove the comment. At the point in time when we start testing in a ci fashion we need to think about how to start the application.